database/sqlite3/sqlite3.go

```diff
@@ -100,13 +102,22 @@ func (m *Sqlite) Open(url string) (database.Driver, error) {
 		return nil, err
 	}
 
-	migrationsTable := purl.Query().Get("x-migrations-table")
+	qv := purl.Query()
+
+	migrationsTable := qv.Get("x-migrations-table")
 	if len(migrationsTable) == 0 {
 		migrationsTable = DefaultMigrationsTable
 	}
+
+	noTxWrap, err := strconv.ParseBool(qv.Get("x-no-tx-wrap"))
+	if err != nil {
+		noTxWrap = false
```

Comment:
It's cleaner to return an error for invalid values. We should handle non-existent (e.g. unspecified) values separately.

Reply:
I did wonder about this, so I went looking for existing patterns.  I found this behaviour in the [cockroachdb driver](https://github.com/golang-migrate/migrate/blob/6dc95bd2010d471d36c59629bce101fc916677e5/database/cockroachdb/cockroachdb.go#L130-L133).  Would you still prefer an error -- at the cost of consistency -- in this case?

Reply:
Yeah, it's cleaner to fail for invalid inputs instead of silently assuming a value

Reply:
I have pushed a commit with the change you requested.  PTAL, @dhui.